### PR TITLE
Add Creative Commons notice

### PR DESCRIPTION
**Pull request by marekweb**
Add a CC notice to the bottom of each post page.

SVG badge image is taken from: https://creativecommons.org/about/downloads/

The terms of use of CC license icons:

> Creative Commons License Buttons and Icons: Creative Commons licenses the use of its button marks that describe a particular legal tool and its icon marks that describe a key license element, such as BY, NC, ND, and SA, on the conditions that you use the mark solely to describe the Creative Commons legal tool that applies to a particular work and, in a manner reasonable to the medium and context, include the URI or hyperlink to the relevant Commons deed on the Creative Commons server.

Source: https://creativecommons.org/policies

### CODE REVIEW

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
I would not draw excess attention to the license, I think it can be faint text without a card. I would prefer it to be on every page (perhaps in `footer` above the links, or at the bottom of `layout`). Here are some examples:

Screen Shot 2021-05-26 at 6 11 45 PM

Screen Shot 2021-05-26 at 6 12 40 PM

**Comment by marekweb** [comment]
Here's an alternative. I moved the notice to the footer.

Screen Shot 2021-05-27 at 09 36 59

File: components/CreativeCommonsNotice.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+const creativeCommonsUrl = 'https://creativecommons.org/licenses/by-nc-sa/4.0/'
+
+const CreativeCommonsLicense = () => (
+    <div className="my-3">
+        <div className="mb-3 text-muted small">
+            This work is licensed under a{' '}
+            <a href={creativeCommonsUrl} className="text-muted">
```
Comment (ltagliaferri) [inline components/CreativeCommonsNotice.tsx:9]: Could we make this link the same style as the rest of the footer?

```
className="text-mutedext-dark text-decoration-none"
```

**Review by ltagliaferri — COMMENTED** [review]
Just one suggestion to keep the links uniform. What do you think?

File: components/CreativeCommonsNotice.tsx (hunk 1 above)
Comment (marekweb) [reply components/CreativeCommonsNotice.tsx]: I implemented this suggestion. I noticed that `text-muted` overrides the effect of `text-dark` so I omitted the latter class.

**Review by ltagliaferri — APPROVED** [review]

**Comment by ltagliaferri** [comment]
I had made a typo in the suggested CSS! I added it in and it doesn't override. Thanks!

**Pull request merged**